apps/web/modules/auth/forgot-password/[id]/forgot-password-single-view.tsx

```diff
@@ -108,7 +108,7 @@ export default function Page({ requestId, isRequestExpired, csrfToken }: PagePro
                   },
                   pattern: {
                     message: "Should contain a number, uppercase and lowercase letters",
-                    value: /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).*$/gm,
+                    value: /^(?=.*\d)(?=.*\p{Ll})(?=.*\p{Lu})(?=.*[\p{Ll}\p{Lu}]).*$/gmu,
```

Comment:
@khushal-malhotra would you kindly care to explain these changes?

Reply:
The \p{Ll} and \p{Lu} pattern in a regex is used to match any lowercase and uppercase letter in Unicode respectively. It will match accented letters from languages like Spanish. This allows for greater inclusivity and flexibility to create unique passwords with characters like 'é', 'ü', and 'ñ' alongside standard English letters.